### DISCUSSION TITLE
Information about @urql/introspection

### DISCUSSION THREAD

**Discussion by innovaweb-dev** [urql-graphql/urql#1595] [Q&A] [answered]
Does exist documentation about it ?
I didn't found this in urql documentation.
https://formidable.com/open-source/urql/docs/api/

**Comment by kitten** [comment, marked as answer]
There's (from our perspective) currently only a single use-case for it, so it's grouped into the "Schema Awareness" page with minimal docs https://formidable.com/open-source/urql/docs/graphcache/schema-awareness/#optimizing-a-schema

**Reply by innovaweb-dev** [reply]
Thanks @kitten 
In my case, I would like to use introspection to get to the user the possibility to select fields from interface and to choose the columns which he wants in his data table.
